rpcserverhelp.go

```diff
@@ -153,11 +153,13 @@ var helpDescsEnUS = map[string]string{
 	// GetBlockCmd help.
 	"getblock--synopsis":   "Returns information about a block given its hash.",
 	"getblock-hash":        "The hash of the block",
-	"getblock-verbose":     "Specifies the block is returned as a JSON object instead of hex-encoded string",
-	"getblock-verbosetx":   "Specifies that each transaction is returned as a JSON object and only applies if the verbose flag is true (btcd extension)",
-	"getblock--condition0": "verbose=false",
-	"getblock--condition1": "verbose=true",
+	"getblock-verbosity":   "Specifies the block format returns",
```

Comment:
The rest of the help uses `command-verbose` and `verbose=n` patterns, not a `command-verbosity` and `verbosity=n`. For example, https://github.com/btcsuite/btcd/blob/ab8fa7bf183473a5c37cd3c5e04e91c9a4f52dc3/rpcserverhelp.go#L463